README.md

```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
 ./bin/clear-data.sh
 ```
 
-
 ### GATSBY!! (or the start of it)
+
 Gotta start somewhere! I'm putting a placeholder header section for now that will further outline how to work with support-docs on Gatsby.
 
+#### How to preview changes
+
+1. Create a branch
+2. Create a PR to master
+3. Go to https://www.gatsbyjs.com/dashboard/organization/60c30363-21fd-4739-a2ca-75f5011a3a1b/sites
+4. Click on the link "Support Docs"
+5. You are now in the "builds" section. here you can see a "production" section and a "pull requests" section
+6. Look in the "pull requests" section for the PR you just created and click "view build"
+
+- This will show a version for the PR current build of the static gatsby site
+
+#### How to deploy changes
+
+1. Complete the steps above in the "How to preview changes" section
+2. Once the PR is approved by a reviewer and the PR is merged and squashed, the build will kick off automatically.
```

Comment:
Do we need to specify the branch for production builds?

Reply:
I personally don't think so since merges to master/main will kick off the build and that's just git knowledge.